src/main/java/ratpack/zipkin/internal/MethodCapturingRequestSpec.java

```diff
@@ -60,7 +67,8 @@ public RequestSpec headers(Action<? super MutableHeaders> action) throws Excepti
 
   @Override
   public RequestSpec method(HttpMethod method) {
-    capturedMethod = method;
+    this.capturedMethod = method;
+    span.set(impl.handler.handleSend(impl.injector, getHeaders(), this));
```

Comment:
I should also have mentioned, I think we need to at the very least start the parent span within an ExecHarness.yield or run method so the span context is populated from a Ratpack managed thread.  I'm not sure we can do this with the current tests.   Unfortunately when I tried that by rewriting one of the tests it still failed :(